src/components/Navbar.tsx

```diff
@@ -0,0 +1,290 @@
+                    height="20"
+                    src="/nav_close.svg"
+                    alt="CLOSE"
+                    className="z-50"
+                    onClick={hideSidebar}
+                  />
+                )}
+              </div>
+              <div className="drawer-side h-[calc(100dvh)] w-screen overscroll-none bg-[#644A93]">
+                <div className="z-50 flex h-full flex-col justify-between px-16">
+                  <div className="z-50 flex w-fit flex-col gap-8 pt-28 text-xl text-white">
+                    <Link href="/" onClick={hideSidebar}>
+                      About
+                    </Link>
+                    <Link href="#Community" onClick={hideSidebar}>
+                      Statistics
+                    </Link>
+                    <Link href="#Carousel" onClick={hideSidebar}>
+                      Past Events
+                    </Link>
+                    <Link href="#Sponsors" onClick={hideSidebar}>
+                      Sponsors
+                    </Link>
+                    <Link href="#FAQ" onClick={hideSidebar}>
+                      FAQ
+                    </Link>
+                  </div>
+                  <div className="align-center flex w-full items-center justify-center gap-4 py-16 md:gap-4">
+                    <Link href="https://www.linkedin.com/company/deltahacks/mycompany/">
+                      <Image
+                        className=""
+                        width="36"
+                        height="36"
+                        src="/nav_linkedin.svg"
+                        alt="logo"
+                      />
+                    </Link>
+                    <Link href="https://www.instagram.com/deltahacks">
+                      <Image
+                        className=""
+                        width="36"
+                        height="36"
+                        src="/nav_insta.svg"
+                        alt="Instagram"
+                      />
+                    </Link>
+                    <Link href="https://www.instagram.com/deltahacks">
+                      <Image
+                        className=""
+                        width="36"
+                        height="36"
+                        src="/nav_devpost.svg"
+                        alt="Devpost"
+                      />
+                    </Link>
+                    <Link href="https://www.tiktok.com/@deltahacks">
+                      <Image
+                        className=""
+                        width="36"
+                        height="36"
+                        src="/nav_tiktak.svg"
+                        alt="TikTok"
+                      />
+                    </Link>
+                  </div>
+                </div>
+              </div>
+            </Drawer.Content>
+          </Drawer.Portal>
+        </Drawer.Root>
+      </div>
+    </div>
+  );
+};
+
+export default Navbar;
+
+//  <div className="drawer drawer-end">
```

Comment:
@demanr Did you want to keep these comments in the push?

Reply:
I'll remove them then

Reply:
Ok that works